Add `cudf.options` (#11193)
rapidsai/cudf: docs/cudf/source/user_guide/options.md

# Options

cuDF has an options API to configure and customize global behavior.
This API complements the [pandas.options](https://pandas.pydata.org/docs/user_guide/options.html) API with features specific to cuDF.

{py:func}`cudf.describe_option` will print the option's description,

> the current and default value.

Reviewer: ```suggestion
the current value, and the default value.
```

When no argument is provided,
all options are printed.

> To set value to a option, use {py:func}`cudf.set_option`.

Reviewer: ```suggestion
To set an option's value, use {py:func}`cudf.set_option`.
```

> See [API reference](api.options) for more details.

Reviewer 2: ```suggestion
See the [API reference](api.options) for more details.
```

# Options

cuDF has an options API to configure and customize global behavior.
This API complements the [pandas.options](https://pandas.pydata.org/docs/user_guide/options.html) API with features specific to cuDF.

User may get the full list of cudf options with {py:func}`cudf.describe_option` with no arguments.

> To set value to a option, use {py:func}`cudf.set_option`.

Author: Actually, this line is not particularly accurate. Now cudf.describe_option will only print the option description, not returning them.
